docs/_includes/segmented-buttons.html

```diff
@@ -0,0 +1,122 @@
+<div id="segmented-buttons">
+    {% include section-header.html name="segmented-buttons" version=page.versions.segmentedButton %}
+
+    <p>Segmented Buttons are non-form toggle buttons used in certain cases. For detailed requirements and when to use these over tabs, please visit the <a href="https://ebay.gitbook.io/mindpatterns/input/segmented-buttons">eBay MIND Pattern for Segmented Buttons</a>. </p>
+    <p>The selected toggle button is toggled by <span class="highlight">aria-current="true"</span> attribute.</p>
+    <p><strong>NOTE:</strong> JavaScript is required to handle segmented buttons currently selected button.</p>
+    <p></p>
+
+    <h3 id="button-default">Default Segmented buttons</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons">
+                <ul class="segmented-buttons__items">
+                    <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button"
+                            aria-current="true">Day</button></li>
+                    <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button">Month</button></li>
+                    <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button">Year</button></li>
+                </ul>
+            </div>
+
+        </div>
+    </div>
+
+    {% highlight html %}
+<div class="segmented-buttons">
+    <ul class="segmented-buttons__items">
+        <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button"
+                aria-current="true">Day</button></li>
+        <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button">Month</button></li>
+        <li class="segmented-buttons__item"><button type="button" class="segmented-buttons__button">Year</button></li>
+    </ul>
+</div>
+    {% endhighlight %}
+
+    <h3 id="button-default">Segmented buttons - with icons</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons">
+                <ul class="segmented-buttons__items">
+                    <li class="segmented-buttons__item">
+                        <button type="button" aria-current="true" class="segmented-buttons__button">
```

Comment:
With every node having a class, it's starting to feel a little like class-itis. I know this is true of other modules too, so it's probably worthy of a broader conversation next time we approach a major release (when we can do breaking changes). Generally I feel like going down to a maximum two levels of descendant selector is fine. Also some nodes need a class if they tend to be manipulated with js a lot.